bitrise.yml

```diff
@@ -0,0 +1,160 @@
+---
+format_version: '8'
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+project_type: ios
+trigger_map:
+- push_branch: develop
+  workflow: deploy_staging
+- push_branch: master
+  workflow: deploy_appstore
+- push_branch: "*"
+  workflow: test
+workflows:
+  deploy_appstore:
+    steps:
+    - activate-ssh-key@4:
+        run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
+    - script@1:
+        inputs:
+        - content: |-
+            #!/bin/bash
+            set -ex
+            RBENV_DIR="`cd $(rbenv which ruby)/../..;pwd`"
+            echo "Gem cache directory: $RBENV_DIR"
+            envman add --key GEM_CACHE_PATH --value $RBENV_DIR
+        title: Set GEM_CACHE_PATH env var
+    - git-clone@6.1: {}
+    - cache-pull@2: {}
+    - cocoapods-install@2: {}
+    - xcode-test@3.0: {}
+    - fastlane-match@0:
+        inputs:
+        - type: appstore
+        - team_id: "$TEAM_ID"
+        - decrypt_password: "$MATCH_PASSWORD"
+        - git_url: "$MATCH_REPO_URL"
+        - app_id: "$BUNDLE_ID"
+    - certificate-and-profile-installer@1: {}
+    - set-xcode-build-number@1:
+        inputs:
+        - plist_path: "$INFO_PLIST_PATH"
+    - xcode-archive@3:
+        inputs:
+        - export_method: app-store
+    - deploy-to-itunesconnect-application-loader@1:
+        inputs:
+        - password: "$APPLE_PASSWORD"
+        - itunescon_user: "$APPLE_ID"
+        - app_password: "$APPLE_APP_SPECIFIC_PASSWORD"
```

Comment:
What do you think if we use [Apple API key](https://devcenter.bitrise.io/getting-started/connecting-to-services/setting-up-connection-to-an-apple-service-with-api-key/) (also [suggested on Bitrise](https://devcenter.bitrise.io/getting-started/connecting-to-services/bitrise-steps-and-their-authentication-methods/#deploy-to-itunes-connect---application-loader-step)). There will be no need to refreshing token so often?